GameData/RealismOverhaul/RO_SuggestedMods/BoringCrewServices/RO_Starliner.cfg

```diff
@@ -0,0 +1,780 @@
+	@mass = 0.23
+
+	!RESOURCE,* {}
+	!MODULE[ModuleAblator] {}
+	!MODULE[ModuleLiftingSurface] {}
+	!MODULE[ModuleB9PartSwitch]:HAS[#moduleID[AblatorSwitch]] {}
+
+	%heatShieldDiameter = 4.56
+	%heatShieldTag = Lunar
+	%resetHeatShieldAblator = true
+	%resetHeatShieldMass = true
+}
+
+
+//	Primary Heat Shield
+@PART[BCS_Centauri_HeatShield]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.6
+	@title = CST-100 Heatshield
+	@manufacturer = #roMfrBoeing
+	@description = Ablative Heatshield for the CST-100 Starliner, rated for LEO Re-entries only.
+	@mass = 0.23
+
+	!RESOURCE,* {}
+	!MODULE[ModuleAblator] {}
+	!MODULE[ModuleLiftingSurface] {}
+	!MODULE[ModuleB9PartSwitch]:HAS[#moduleID[AblatorSwitch]] {}
+
+	%heatShieldDiameter = 4.56
+	%heatShieldTag = LEO
+	%resetHeatShieldAblator = true
+	%resetHeatShieldMass = true
+}
+
+
+//	RS-88 Abort Engines
+@PART[BCS_Centauri_LAE]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.6
+	!MODULE[ModuleB9PartSwitch]:HAS[#moduleID[engineSwitch]] {}
+	MODULE
+	{
+		name = ModuleB9PartSwitch
+		moduleID = engineSwitch
+		switcherDescription = Type
+		switcherDescriptionPlural = Types
+		SUBTYPE
+		{
+			name = Abort
+			transform = standardNozzle
+		}
+		SUBTYPE
+		{
+			name = AbortRS88
+			transform = standardNozzle
+		}
+		// SUBTYPE
+		// {
+		// 	name = Orbital
+		// 	transform = vacNozzle
+		// }
+	}
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = vacNozzle
+	}
+	%engineType = RS88
+}
+
+
+//	Main Chutes
+@PART[BCS_Centauri_MainChute]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.6
+	@CoMOffset = -0.627468, 0, 0
+	@title = CST-100 Main Chutes
+	@manufacturer = #roMfrBoeing
+	@description = A trio of large parachutes to ensure a safe return to Earth for the Starliner Command Module. The swing arm centers the parachutes at the end of full deployment for a comfortable touchdown attitude.
+	%skinTempTag = Fiberglass
+	%internalTempTag = Instruments
+	!MODULE[ModuleParachute],* {}
+	MODULE:NEEDS[RealChute]
+	{
+		name = RealChuteModule
+		caseMass = 0.1
+		timer = 0
+		mustGoDown = false
+		spareChutes = 1
+		cutSpeed = 0.25
+
+		PARACHUTE
+		{
+			parachuteName = canopy
+			capName = Cap
+			preDeploymentAnimation = semi_deploy
+			deploymentAnimation = full_deploy
+			material = Nylon
+			minIsPressure = False
+			minPressure = 0.01
+			preDeployedDiameter = 16
+			preDeploymentSpeed = 3.0
+			minDeployment = 2895
+			deployedDiameter = 48
+			deploymentSpeed = 5.0
+			deploymentAlt = 2000
+			cutAlt = 0
+		}
+	}
+}
+
+
+//	Docking Cover
+@PART[BCS_Centauri_NDS_cover]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.6
+	@title = CST-100 Docking Port Cover
+	@manufacturer = #roMfrBoeing
+	@description = A hingable cover for the NASA Docking System port on the front end of Boeing's Starliner capsule, designed to shield the docking port when not attached to the Space Station.
+	%skinTempTag = HRSI
```

Comment:
Do these parts want or need to be HRSI?

Reply:
Part is made of and covered with the same insulation material as the rest of the capsule.

It doesn’t _need_ to be HRSI to survive re-entry but it logically makes sense.

Reply:
This article is pretty old (2013) but claims that the CST-100 backshell TPS is a mixture of BRI-18 (HRSI) and FRSI blankets. Looking at actual pictures of Starliner, it's pretty obvious that the white parts of the backshell are FRSI blankets, the only HRSI is the black tiles around the base of the heat shield and hatches,